include/zephyr/dt-bindings/ethernet/dsa_tag_proto.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2025 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_DSA_TAG_PROTO_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_DSA_TAG_PROTO_H_
+
+#define DSA_TAG_PROTO_NOTAG	0
+#define DSA_TAG_PROTO_NETC	1
```

Comment:
Some comments describing the defines would be helpful.

Reply:
Thanks. Will add that.